spec/lib/eventbrite/event_spec.rb

```diff
@@ -0,0 +1,27 @@
+require "eventbrite/event"
+require "eventbrite/venue"
+
+describe Eventbrite::Event do
+  describe "#dates" do
+    it "returns the start and end dates as a single date format" do
+      details = {
+        "start" => { "local" => "2014-09-19" },
+        "end" => { "local" => "2014-09-20" },
+      }
+      event = Eventbrite::Event.new(details)
+
+      expect(event.dates).to eq("September 19 & 20, 2014")
+    end
+  end
+
+  describe "#venue" do
+    it "returns the venue of the event" do
+      venue = double(:venue)
+      allow(Eventbrite::Venue).to receive(:new).and_return(venue)
```

Comment:
This mocking doesn't seem to be really doing anything. Particularly the details part.

How about either mocking out that it should receive the details hash, or just assert that the method returns an instance of the `Eventbrite::Venue` class?